docs/operator-guides/configuring-connector-resources.md

```diff
@@ -4,23 +4,43 @@ products: oss-*
 
 # Configuring Connector Resources
 
-There are four different types of jobs—SYNC, CHECK, DISCOVER and SPEC.
+There are four different types of jobs: SYNC, CHECK, DISCOVER and SPEC. Although it is possible to configure resources for all four jobs, we focus on Sync jobs as it is the most frequently run job.
 
-Although it is possible to configure resources for all four jobs, we focus on Sync jobs as it is the most frequently run job.
+There are three different ways to configure connector resource requirements for a sync. The narrower in scope the requirement, the higher the precedence.
 
-There are three different ways to configure connector resource requirements for a Sync:
+1. Instance-wide: applies to all containers in a sync. Lowest precedence. Overridden by all other configuration. Intended to be a default.
+2. Connector-specific: applies to all containers of that connector type in a sync. Second-highest precedence. Overrides instance-wide configuration.
+3. Connection-specific: applies to all containers of that connection in a sync. Highest precedence. Overrides all other configuration.
```

Comment:
There are four in SME as of 1.5:

1. instance-wide
2. actor-definition-specific: IE: All Postgres sources
3. actor-specific: IE: This specific Postgres source I set up (this is the new one)
4. connection-specific

Reply:
I am not sure how we externally communicate actor definition vs. actor, but I think in this docs update it makes sense to be more specific